File: AirportManagement.API/AirportManagement.API/Models/Airport.cs
using System;
namespace AirportManagement.API.Models
{
	public class Airport
	{
		public int Id { get; set; }

		public string AirportName{ get; set; }

		public string Country { get; set; }

		public string State { get; set; }

		public string City { get; set; }

		public DateTime CreatedAt { get; set; }

		public DateTime UpdatedAt { get; set; }

		public ICollection<Flights>? ArrivingFlights { get; set; }

        public ICollection<Flights>? DepartureFlights { get; set; }
    }
}

File: AirportManagement.SPA/src/app/_models/Airport.ts
import { Flight } from "./Flight";

export interface Airport {
    Id: number;
    AirportName: string;
    Country: string;
    State: string;
    City: string;
    CreatedAt: string;
    UpdatedAt: string;
    ArrivingFlights: Flight[];
    DepartureFlights: Flight[]
}
